spring-cloud-starter-task-composedtaskrunner/src/main/java/org/springframework/cloud/task/app/composedtaskrunner/StepBeanDefinitionRegistrar.java

```diff
@@ -113,7 +113,7 @@ private ComposedTaskProperties composedTaskProperties() {
 		}
 		if (dataFlowUriString != null) {
 			try {
-				properties.setDataFlowUri(new URI(dataFlowUriString));
+				properties.setDataflowServerUri(new URI(dataFlowUriString));
```

Comment:
Is there a reason why we set this default in the constructor rather than on the field itself like the other default values?